src/index.ts

```diff
@@ -274,53 +253,39 @@ export async function getLibreLinkUpConnection(): Promise<string | null>
     }
 }
 
-async function lastEntryDate(): Promise<Date | null>
-{
-    const url = getNightscoutUrl() + "/api/v1/entries?count=1"
-    const response = await axios.get(
-        url,
-        {
-            headers: nightScoutHttpHeaders
-        });
-
-    if (!response.data || response.data.length === 0)
-    {
-        return null;
-    }
-    return new Date(response.data.pop().dateString);
+let nightscoutClient: NightscoutAPI;
+if (process.env.NIGHTSCOUT_API_V3 === "true") {
+    nightscoutClient = new ClientV3();
+} else {
+    nightscoutClient = new ClientV1(
```

Comment:
What if the token is not provided?

Reply:
I suggest to add parameters sanitation and validation here.

Reply:
Done